src/common/gitProvider/azureDevops.ts

```diff
@@ -476,7 +520,16 @@ ${this.getPipelineVariablesConfig()}
     const repositoryId = process.env.BUILD_REPOSITORY_ID || null;
     const buildId = process.env.BUILD_BUILD_ID || null;
     const jobId = process.env.SYSTEM_JOB_ID || null;
-    const pullRequestIdStr = getEnvVar("SYSTEM_PULLREQUEST_PULLREQUESTID") || prInfo?.idStr || null;
+    let pullRequestIdStr = getEnvVar("SYSTEM_PULLREQUEST_PULLREQUESTID") || prInfo?.idStr || null;
```

Comment:
@blaissaoui we don't need this part, as getpullRequestInfo is called before, and it already contains a call to extractPullRequestIdFromCommitMessage , right ?